omod/src/main/webapp/pages/results.gsp

```diff
@@ -60,7 +76,39 @@
             iDisplayLength: 10,
             sPaginationType: "full_numbers",
             bAutoWidth: false,
-            sDom: 't<"fg-toolbar ui-toolbar ui-corner-bl ui-corner-br ui-helper-clearfix datatables-info-and-pg "ip>'
+            sDom: 'Bt<"fg-toolbar ui-toolbar ui-corner-bl ui-corner-br ui-helper-clearfix datatables-info-and-pg "ip>',
+            buttons: [
+                {
+                    extend: 'excelHtml5',
+                    text: '<i class="icon-file-excel" title="Export as Excel"></i>',
+                    className: 'export-btn',
+                    exportOptions: {
+                        columns:[ 3,1,2,4 ]
```

Comment:
@shivtej1505 Can the columns to be exported be defined as a single variable to reduce duplication since the same order will always be exported